motor/providers/terraform/provider.go

```diff
@@ -79,22 +85,36 @@ func New(tc *providers.Config) (*Provider, error) {
 		}
 
 		if stat.IsDir() {
-			fileList, err := os.ReadDir(path)
-			if err != nil {
-				return nil, err
-			}
-
-			err = loader.ParseHclDirectory(path, fileList)
-			if err != nil {
-				return nil, errors.Wrap(err, "could not parse hcl files")
-			}
-
-			err = ReadTfVarsFromDir(path, fileList, tfVars)
-			if err != nil {
-				return nil, errors.Wrap(err, "could not parse tfvars files")
-			}
-
-			modulesManifest, err = ParseTerraformModuleManifest(path)
+			filepath.WalkDir(path, func(path string, d fs.DirEntry, err error) error {
+				if err != nil {
+					return err
+				}
+				foundExamples := MODULE_EXAMPLES.FindString(path)
```

Comment:
I'd like to have opinions on this one. I don't think it makes sense when we scan example file like they are included in modules: `mondoo-operator/.github/terraform/aws/.terraform/modules/vpc/examples/secondary-cidr-blocks/main.tf`

Should we perhaps skip more?

Reply:
So from what I read `examples` is a standard folder in terraform. Would there be any reason why someone would ever want to scan the content of that folder?

Reply:
I don't know why someone would scan these. When they use the examples, the code will end up in a place where is relevant and scanned.